logos/src/logos/capacity/capacity_planner.py

```
@@ -1951,7 +2155,47 @@ def _compute_demand_actions(
+            # lane(s) would use. Models targeting disjoint GPUs proceed.
+            #
+            # For COLD models with no existing lane the GPU set is unknown
+            # here (model_lanes is empty), and the empty frozenset would be
+            # treated by has_overlapping_reservation as "match anything" —
+            # which would incorrectly block every cold seed/load whenever
+            # any unrelated reservation is in flight. Defer that check to
+            # the cold-load branch below where we have a concrete placement
+            # GPU set from _pick_cold_load_placement.
+            if self._eviction_gate_v2 and committed > 0 and model_lanes:
+                target_gpus: frozenset[int] = frozenset()
+                for l in model_lanes:
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)